### PR TITLE
Test for bud / tag:

### PR DESCRIPTION
**Pull request by naude-r**
- add a test for bud --build-arg
- add a test for buildah tag

### CODE REVIEW

**Comment by naude-r** [comment]
@rohanKanojia tests cases for:
- bud --build-arg
- tag command

**Comment by naude-r** [comment]
@rohanKanojia would it be possible to make a new release?

**Comment by rohanKanojia** [comment]
@naude-r :Sure, I'll try to cut a release sometime tomorrow

File: buildah-core/src/test/java/io/jshift/buildah/core/commands/BudCommandTest.java [hunk 1]
```diff
@@ -29,4 +31,17 @@ public void should_execute_bud_command() {
         assertThat(cliCommand).containsExactly(transform("bud --tls-verify=false --file Dockerfile --file Dockerfile.fed -t targetImg dockerFileContext"));
 
     }
+
+    @Test
+    public void should_add_build_arg() {
+
+        Map<String, String> buildArgs = new HashMap<>();
```
Comment (manusa) [inline buildah-core/src/test/java/io/jshift/buildah/core/commands/BudCommandTest.java]: Should be possible to replace by:
`Map<String, String> buildArgs = Collections.singletonMap("TEST", "TEST_VAL");`

File: buildah-core/src/test/java/io/jshift/buildah/core/commands/BudCommandTest.java [hunk 2]
```diff
@@ -29,4 +31,17 @@ public void should_execute_bud_command() {
...
+        buildArgs.put("TEST", "TEST_VAL");
+        List<String> dockerfileList = new ArrayList<>();
```
Comment (manusa) [inline buildah-core/src/test/java/io/jshift/buildah/core/commands/BudCommandTest.java]: Should be possible to replace by:
`List<String> dockerfileList = Collections.singletonList("Dockerfile");`

File: buildah-core/src/test/java/io/jshift/buildah/core/commands/TagCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+@ExtendWith(MockitoExtension.class)
+public class TagCommandTest {
+
+    @Mock
+    CliExecutor buildahExecutor;
+
+    @Test
+    public void should_execute_tag_command() {
+
+        List<String> tags = new ArrayList<>();
```
Comment (manusa) [inline buildah-core/src/test/java/io/jshift/buildah/core/commands/TagCommandTest.java]: Should be possible to replace by:
`List<String> tags = Arrays.asList("latest", "v1.0.0");`

**Review by manusa — CHANGES_REQUESTED** [review]
Please check comments and squash your changes in a single commit

**Comment by naude-r** [comment]
> Please check comments and squash your changes in a single commit

comments addressed. have an issue when attempting to squash the commits. any hints?

**Comment by naude-r** [comment]
> > Please check comments and squash your changes in a single commit
> 
> comments addressed. have an issue when attempting to squash the commits. any hints?

nevermind. got it working....

**Review by manusa — APPROVED** [review]
LGTM, thanks for your contribution! :wink:

**Review by rohanKanojia — APPROVED** [review]

**Comment by rohanKanojia** [comment]
@naude-r: I've released 1.0.1 which contains this fix

**Comment by naude-r** [comment]
@rohanKanojia Thank you!. Working just fine.

**Pull request merged**